spec/profiles/profile_spec.py

```diff
@@ -555,6 +556,23 @@ def get_range_error():
             # [!] Energy must match
             assert total_expected == total_estimated, "For tariff '{}' Total energy '{}' must match the expected '{}'".format(a_tariff["tariff"], total_estimated, total_expected)
 
+    with it("3.1A_LB"):
+        kwargs = {'LB': True, 'kva': 1}
+        the_tariff = T31A(**kwargs)
+        balance = {
+            'P1': 100,
+            'P2': 80,
+            'P3': 60,
+            'P5': 15,
+            'P6': 15
+        }
+        periods = the_tariff.energy_periods
+        total_expected = sum(balance.values())
+
+        estimation = self.profile.estimate(the_tariff, balance)
+        total_estimated = sum([x.measure for x in estimation.measures])
+        assert total_expected != total_estimated, "3.1A_LB"
```

Comment:
Maybe here you could assert the expected amount of profiled energy

Reply:
Doned